gateway-api/users-api/api/get-user.spec.js

```diff
@@ -0,0 +1,17 @@
+const app = require('../app.js');
+const should = require('should');
+const supertest = require('supertest');
+
+describe('/get-user', () => {
+    it('should return a users individual data', done => {
+      supertest(app)
+      .get('/get-user')
+      .end((err, res) => {
+        res.text.should.equal(
```

Comment:
This is a syntax error. To run your tests, run `npm test` from within the users-api folder.

Also, validations should look like this:

```javascript
res.body.username.should.equal('bob');
```